tests/test_memory_store.py

```
@@ -124,14 +124,82 @@ def test_get_stats(self):
+                    # Load existing data
+                    worker_memory.load()
+
+                    # Add a new event
+                    embedding = np.random.rand(384)
+                    metadata = {'severity': 0.5, 'type': f'process_{process_id}_event'}
+                    worker_memory.write(embedding, metadata)
+
+                    # Save the updated data
+                    worker_memory.save()
+
+                    return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)